recipes/otgu/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.0" %}
+
+package:
+  name: otgu
+  version: {{ version }}
+
+source:
+  fn: otgu-{{ version }}.tar.gz
+  url: https://github.com/openturns/otgu/archive/v{{ version }}.tar.gz
+  sha256: 001d5cf8445c6fc665d728c7cea8e475e3d9e2b7ec363cbd398c0faaadea2c96 
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('otgu', max_pin='x.x') }}
+
+requirements:
+  build:
+    - cmake  # [not win]
+    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('cxx') }}  # [not win]
+  host:
+    - python
+    - swig  # [not win]
+    - openturns
+  run:
+    - python
+    - openturns
+
+test:
+  imports:
+    - otgu
+
+about:
+  home: http://openturns.org/
+  license: LGPL3
+  license_file: COPYING.LESSER
```

Comment:
Please add a `license_family` entry.  CC @conda-forge/core this would have been caught by https://github.com/conda-forge/conda-smithy/pull/1256